packages/workbox-background-sync/Queue.mjs

```diff
@@ -132,17 +134,17 @@ class Queue {
       replayedRequests.push(replay);
     }
 
+    await this._runCallback('queueDidReplay', replayedRequests);
+
     // If any requests failed, put the failed requests back in the queue
-    // and register for another sync.
+    // and reject promise.
```

Comment:
Maybe reword this to: "and rethrow the failed requests"?

Note: this comment I'm less concerned about changing...

Reply:
Done